src/components/Intro.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import Jumbotron from 'react-bootstrap/Jumbotron';
+import Button from 'react-bootstrap/Button';
+
+class Intro extends Component 
+{
+    state = {  }
+    render() 
+    { 
+        return ( 
+        <div>
+            <Jumbotron>
+                <h1 >Welcome to Gopher A Bite!</h1>
+                <p>
+                    Are you a University of Minnesota, Twin Cities student? Do you love food? We'll you're in the right place! 
```

Comment:
We'll -> Well. Maybe put 'Meet the Creators' as a separate tab? The Jumbotron component stands out as white while the reviews component and every other component is yellow.